amy/workshops/migrations/0244_auto_20210509_2051.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.18 on 2021-05-09 20:51
+
+from django.db import migrations, models
+
+
+def find_duplicate_members(apps, schema_editor):
+    """The next migration will enforce a unique constraint in DB, which may result in
```

Comment:
Since this only alerts on duplicates, is it somehow handled manually? How are the duplicates addressed?

Reply:
Migration step will fail and the code upgrade will not work. This will probably happen on test server, but will not happen in production server.

It requires human intervention - someone who can assess and remove the duplicates.